addressing/README.md

```diff
@@ -0,0 +1,43 @@
+# Addressing on the Decentralized Web
+
+> Note: This document is work in progress and largely incomplete.
+
+Authors:
+- Lars Gierth \<lgierth@ipfs.io>
+
+Table of contents:
+- Introduction
+- The precarious web
+  - Link competition and link rot
+  - The addressing rift
+- DWeb Addressing
+- Namespaces
+  - /ipfs -- immutable data
+  - /ipns -- mutable pointers
+  - Addressing data from other content-addressed systems
+  - Network addressing
+- Interoperability
+  - DWeb Addressing with HTTP
+  - URL schemes
```

Comment:
I would explicitly put names here ("URL schemes: `ipfs://` and `ipns://`"), just like we have `dweb:` below.
It is what people usually ask about, and it makes sense to have these names pop.